src/main/java/com/launchdarkly/client/LDUser.java

```diff
@@ -248,62 +272,72 @@ public Builder email(String email) {
     }
 
     /**
-     * Add a {@link java.lang.String}-valued custom attribute
-     * @param k the key for the custom attribute
+     * Add a {@link java.lang.String}-valued custom attribute.
+     *
+     * @param k the key for the custom attribute. When set to one of the built-in user attribute keys, this custom attribute will be ignored.
```

Comment:
Perhaps consider referencing the documentation directly on which keys are considered "built-in" (http://docs.launchdarkly.com/docs/targeting-users#targeting-based-on-user-attributes)?

Given that `UserAttribute` is package private it would be good to have an explicit reference to all attributes are considered "built in".